content/en/apps/examples/anc.md

```diff
@@ -94,6 +101,125 @@ Both maternal and newborn care workflows are defined to connect form actions and
+         resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+         %}}
+  {{% workflow
+         condition="The child assessment form indicates that the patient has a cough less than 14 days."
+         task="Two child treatment follow up tasks are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
+         resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+         %}}
+  {{% workflow
+         condition="The child assessment form indicates that the patient has a cough less than 14 days and a fast breathing."
+         task="Two child treatment follow up tasks are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
+         resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+         %}}
+   {{% workflow
+         condition="The child assessment form is submitted with a diarrhea that lasts less than 14 days and with no blood."
+         task="Two child treatment follow up tasks are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
+         resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+         %}}
+  {{% workflow
+         condition="The child assessment form is submitted with a yellow MUAC."
+         task="Two child treatment follow up tasks are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
+         resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+  %}}
+{{% /workflow-table %}} 
+
+## Supplement schedule
+
+{{% schedule %}}
+|| 6 mo | 12 mo | 18 mo | 2 yr | 2.5 yr | 3 yr | 3.5 yr | 4 yr | 4.5 yr | 5 yr |
+|------------|--|--|--|--|--|--|--|--|--|--|
+| Deworming  | | 🟢  | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 |
+| Vitamin A  | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 | 🟢 |
+{{% /schedule %}}
+
+## Immunization schedule
+
+||birth | 6 wk | 10 wk | 14 wk| 6 mo | 9 mo | 12 mo | 18 mo |         
+|------------|--|--|--|--|--|--|--|--|--|--|        
+| BCG  |🟢 |   |  |  |  |  |  |  |  | |       
+| Oral Polio Vaccine (OPV)  | 🟢 | 🟢 | 🟢 | 🟢 |  |  |  |  |  |  |       
+| RotaVirus vaccine  | | 🟢 | 🟢 |  |  |  |  |  |  |  |        
+| Pneumococcal vaccine  |  | 🟢 | 🟢 | 🟢 |  |  |  |  | |  |       
+| Pentavalent vaccine  |  | 🟢 | 🟢 | 🟢 |  |  |  |  |  |  |        
+| Inactivated Polio Vaccine (IPV)  |  |  |  | 🟢 |  |  |  |  |  |  |        
+| Measles vaccine  |  |  |  |  |  | 🟢 |  | 🟢 |  |  |        
+| Yellow fever vaccine  |  |  |  |  |  | 🟢 |  |  |  |  |       
+
+
```

Comment:
@maremegaye  - I was testing a fix for #424 and was using your tables in this PR to make sure my fixes worked. I noticed this table had too many columns on the right.  The table headers (first row) has 9 columns, so all other rows  need to have 9 as well.  You can also add some spaces to visually line up markdown in Sublime.  Here's how I did it, adding spaces and ensuring there's only 9 columns per row:

```markdown
||birth | 6 wk | 10 wk | 14 wk | 6 mo | 9 mo | 12 mo | 18 mo |         
|-----------------------------------|---|---|---|---|--|---|--|---|        
| BCG                               | 🟢 |   |   |   |  |   |  |   |       
| Oral Polio Vaccine (OPV)          | 🟢 | 🟢 | 🟢 | 🟢 |  |   |  |   |       
| RotaVirus vaccine                 |   | 🟢 | 🟢 |   |  |   |  |   |        
| Pneumococcal vaccine              |   | 🟢 | 🟢 | 🟢 |  |   |  |   |       
| Pentavalent vaccine               |   | 🟢 | 🟢 | 🟢 |  |   |  |   |        
| Inactivated Polio Vaccine (IPV)   |   |   |   | 🟢 |  |   |  |   |        
| Measles vaccine                   |   |   |   |   |  | 🟢 |  | 🟢 |        
| Yellow fever vaccine              |   |   |   |   |  | 🟢 |  |   |
```

(when you copy and paste this into Sublime I think it will all line up perfectly!)